.github/workflows/publish_python_sdk.yml

```diff
@@ -23,17 +25,18 @@ jobs:
       - name: Build Package
         run: |
           pip install build && cd python && python -m build
-
-  pypi-publish:
-    runs-on: ubuntu-latest
-    needs:
-      - build-release
-    permissions:
-      id-token: write
-
-    steps:
       - name: Publish release distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           packages-dir: python/dist/
-
+      - name: Send slack message
+        id: slack
+        uses: slackapi/slack-github-action@v1.25.0
+        with:
+          # This data can be any valid JSON from a previous step in the GitHub Action
+          payload: |
+            {
+              "text": "Headsup - The python-sdk has been version bumped and published to pypi",
```

Comment:
Followup PR, can we also include the version # extracted from pyproject? If not that's ok. Also let's link to the pypi: https://pypi.org/project/python-aiconfig/

Reply:
sounds good, will do on top sometime